phlop/run/perf.py

```
@@ -37,7 +38,7 @@ def get_from_files(cli_args):
         with open(path, "r") as file:
             while line := file.readline():
                 test_case = tc.determine_cores_for_test_case(
-                    tc.TestCase(cmd=perf_stat_cmd(cli_args, path, line.rstrip()))
+                    Job(cmd=perf_stat_cmd(cli_args, path, line.rstrip()))
```

Pylint (4.0.6)

[error] No value for argument 'options' in function call

(E1120)